framework/src/systems/NonlinearSystem.C

```diff
@@ -91,8 +91,8 @@ compute_postcheck(const NumericVector<Number> & old_soln,
 
 NonlinearSystem::NonlinearSystem(FEProblemBase & fe_problem, const std::string & name)
   : NonlinearSystemBase(
-        fe_problem, fe_problem.es().add_system<TransientNonlinearImplicitSystem>(name), name),
-    _transient_sys(fe_problem.es().get_system<TransientNonlinearImplicitSystem>(name)),
+        fe_problem, fe_problem.es().add_system<NonlinearImplicitSystem>(name), name),
+    _nl_implicit_sys(fe_problem.es().get_system<NonlinearImplicitSystem>(name)),
```

Comment:
So using `System` will be in the near future?

Reply:
Correct! That one's on @fdkong :)